src/gui/speedlimitdialog.cpp

```diff
@@ -37,10 +39,23 @@ SpeedLimitDialog::SpeedLimitDialog(QWidget *parent)
 {
     m_ui->setupUi(this);
 
-    // Connect to slots
-    connect(m_ui->bandwidthSlider, &QSlider::valueChanged, this, &SpeedLimitDialog::updateSpinValue);
-    connect(m_ui->spinBandwidth, qOverload<int>(&QSpinBox::valueChanged)
-            , this, &SpeedLimitDialog::updateSliderValue);
+    m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow_off")), this, 16));
```

Comment:
```suggestion
    m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow_off")), this, style()->pixelMetric(QStyle::PM_SmallIconSize, nullptr, this)));
```

or maybe a better way is the icon into group box text rather

Reply:
I'll change it when i fix that dialog.